pkgs/development/python-modules/python-kadmin-rs/default.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  pkgs,
+  fetchFromGitHub,
+  rustPlatform,
+  pythonImportsCheckHook,
+  buildPythonPackage,
+
+  cargo,
+  rustc,
+  pkg-config,
+  sccache,
+  setuptools,
+  setuptools-rust,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "python-kadmin-rs";
+  version = "0.4.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "authentik-community";
+    repo = "kadmin-rs";
+    rev = "kadmin/version/${version}";
+    hash = "sha256-Ssv6Qm5JF0yhZd6CKwLIttEtlcH+HfYrDaO171uTMdg=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    hash = "sha256-5pVA+PDFq0utg8v26ch5V45l2AkOSETdTUSb/qIzT54=";
+  };
+
+  buildInputs = [
+    pkgs.krb5
+  ];
+
+  nativeBuildInputs = [
+    sccache
+    pkg-config
+    rustPlatform.bindgenHook
+    pythonImportsCheckHook
+  ];
+
+  build-system = [
+    setuptools
+    setuptools-rust
+    setuptools-scm
+    rustPlatform.cargoSetupHook
+    cargo
+    rustc
```

Comment:
Those 3 should probably go to nativeBuildInputs https://github.com/authentik-community/kadmin-rs/blob/main/pyproject.toml#L41